pages/App.js

```diff
@@ -13,16 +13,62 @@ export default function List() {
     },
     {
       id: nanoid(),
-      text: "Hafermilch und Obst kaufen",
+      text: "die Welt retten",
+    },
+    {
+      id: nanoid(),
+      text: "Blumen gießen",
     },
   ]);
 
+  function addItem(event) {
+    event.preventDefault();
+    const inputElement = event.target.elements.newItem;
+    const newListItem = {
+      id: nanoid(),
+      text: inputElement.value,
+    };
+
+    const newListArray = [...todoItems, newListItem];
+    setTodoItems(newListArray);
+  }
+
+  function deleteItem(id) {
+    const newListArray = todoItems.filter((item) => {
+      if (item.id === id) {
+        return false;
+      } else {
+        return true;
+      }
+    });
+    setTodoItems(newListArray);
+  }
+
   return (
     <>
       <h1>Todo-List</h1>
-      <ul>
+      <form onSubmit={addItem}>
+        <input
+          type="text"
+          id="newItem"
+          name="newItem"
+          placeholder="todo hinzufügen"
+        ></input>
+        <button type="submit">Hinzufügen</button>
+      </form>
+      <ul style={{ listStyle: "none" }}>
         {todoItems.map((item) => (
-          <li key="{item.id}">{item.text}</li>
+          <li key="{item.id}">
+            <input type="checkbox" />
+            {item.text}{" "}
+            <button
+              onClick={() => {
+                deleteItem(item.id);
+              }}
+            >
+              X
+            </button>
+          </li>
```

Comment:
Nice and Short Code. Next cool step is, to refactoring your Code. so divide into compnents. this makes your code even clearer than it already is ;)